src/private/Utils_p.h

```diff
@@ -126,7 +126,7 @@ inline bool usesUtilityWindows()
 
 inline bool isNormalWindowState(Qt::WindowStates states)
 {
-    return !(states & Qt::WindowMaximized) && !(states & Qt::WindowFullScreen);
+    return !(states & (Qt::WindowMaximized | Qt::WindowMinimized | Qt::WindowFullScreen));
```

Comment:
not sure about this one

Being minimized is orthogonal to maximized/normal

Reply:
reverted